lib/eth/contract/function.rb

```diff
@@ -43,7 +43,7 @@ def initialize(data)
     # @param inputs [Array<Eth::Contract::FunctionInput>] function input class list.
     # @return [String] function string.
     def self.calc_signature(name, inputs)
-      "#{name}(#{inputs.collect { |x| x.raw_type }.join(",")})"
+      "#{name}(#{inputs.collect { |x| x.parsed_type.to_s }.join(",")})"
```

Comment:
Just for consistency in this codebase and in Ruby in general. `collect` and `map` are aliases, but `collect` is becoming more and more rare.

```suggestion
      "#{name}(#{inputs.map { |x| x.parsed_type.to_s }.join(",")})"
```